packages/manager/src/views/WebPropertyEnvPage/WebPropertyEnvPage.tsx

```diff
@@ -63,26 +75,36 @@ function getExpiryDayDiff(expiry: string) {
   return `${Math.ceil(dateDiff)}d`;
 }
 const URL_LENGTH_LIMIT = 25;
+
 export const WebPropertyEnvPage = (): JSX.Element => {
   const { query } = useRouter();
   const propertyIdentifier = query.propertyIdentifier as string;
   const { data: session } = useSession();
   const formatDate = useFormatDate();
 
-  const envList = useGetEnvList(propertyIdentifier);
-  const createEnv = useAddEnv(propertyIdentifier);
-  const apiKeys = useGetApiKeys(propertyIdentifier);
-  const createAPIKey = useCreateAPIKey(propertyIdentifier);
-  const deleteAPIKey = useDeleteAPIKey(propertyIdentifier);
+  const envList: any = useGetEnvList(propertyIdentifier);
```

Comment:
Kindly remove all these any, As everything is typed

Reply:
done